proxy/core/src/main/scala/io/cloudstate/proxy/AkkaActorRegisterFeature.scala

```diff
@@ -0,0 +1,33 @@
+package io.cloudstate.proxy
+
+import com.oracle.svm.core.annotate.AutomaticFeature
+import org.graalvm.nativeimage.hosted.Feature.QueryReachabilityAccess
+import org.graalvm.nativeimage.hosted.{Feature, RuntimeReflection}
+
+import scala.collection.JavaConverters._
+
+@AutomaticFeature
+final class AkkaActorRegisterFeature extends Feature {
+
+  override def duringAnalysis(access: Feature.DuringAnalysisAccess) =
+    reachableSubtypes(access, access.findClassByName("akka.akka.Actor")).foreach { cls =>
```

Comment:
Perhaps instead of this, we should instead check invocations of Props-creation, and then add *those* Actors to the reflection configuration?

Reply:
That would be good, but I don't see any public API in Features for doing that.  We could create our own bytecode traversing and recording, but I'm not sure if that's a good idea.

Reply:
@dwijnand Would it be possible to do in Substitutions somehow?

Reply:
I don't see how.